_templates/sidebar_links.html

```diff
@@ -75,7 +71,17 @@
     href="https://github.com/ipython/ipython/wiki/Dev:-Index">Development information</a></li>
     <li><a class="reference external" href="http://travis-ci.org/#!/ipython/ipython">
         Travis CI</a></li>
-  </ul>          
+  </ul>
+</div>
+<div class="sidebarblock">
```

Comment:
-1 on adding a new block to the sidebar, there's already more there than I'd like. And 'Other ...' sections on the homepage seem a bit weak, like we can't decide whether we want these things or not. I know that's because we *haven't*, but we should decide rather than making compromises like this.